lazy_theta_star_planner/package.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>lazy_theta_star_planner</name>
+  <version>0.0.1</version>
+  <description>TODO: Package description</description>
+  <maintainer email="Anshu-man567@todo.todo">anshu-man</maintainer>
```

Comment:
Adding yourself + me would be good. A description would be nice too, even just a few words "lazy theta* planner plugin"

Reply:
Ahhh! Forgot about this part again